.github/workflows/ciwheel.yml

```diff
@@ -67,8 +67,6 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Set up Python
-        uses: actions/setup-python@v2
```

Comment:
Why is this removed? I'd feel more comfortable with explicit versions everywhere.

Reply:
I removed it everywhere where version should not matter.